nodes/__init__.py

```
@@ -23,4 +23,15 @@
 except Exception as e:
     logging.error(f"[MinimaxH3] Failed to import 3D node: {e}")
 
+# H3-aware refinement handoff: legacy API for compatibility with the package entrypoint.
+try:
+    from .minimax_h3_refine_handoff import (
+        NODE_CLASS_MAPPINGS as NODE_CLASS_MAPPINGS_HANDOFF,
+        NODE_DISPLAY_NAME_MAPPINGS as NODE_DISPLAY_NAME_MAPPINGS_HANDOFF,
+    )
+    NODE_CLASS_MAPPINGS.update(NODE_CLASS_MAPPINGS_HANDOFF)
+    NODE_DISPLAY_NAME_MAPPINGS.update(NODE_DISPLAY_NAME_MAPPINGS_HANDOFF)
+except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)